docker/README

```diff
@@ -0,0 +1,14 @@
+This directory is copied up to the container, and is set as the WORKDIR.
```

Comment:
As per IRC, maybe we can use a [docker volume](https://docs.docker.com/engine/userguide/dockervolumes/) instead to save copying. Source trees can get really big. If we use `-v` rather than a hard-coded path in the Dockerfile, we can even serve prod's needs, giving us a decoupled way to provide source to the container after the host-dwelling Jenkins updates it.